src/tools/rbd_mirror/ImageSync.cc

```diff
@@ -157,6 +166,8 @@ void ImageSync<I>::send_copy_image() {
     return;
   }
 
+  update_progress("COPY_IMAGE");
```

Comment:
It would be nice to pass a wrapped ```ProgressContext``` to the ```ImageCopyRequest``` so it can update on the actual image copy (which might take a while).  It should be throttled -- so perhaps the ```ProgressContext``` callback should take a boolean argument for "should_throttle" (or similar).